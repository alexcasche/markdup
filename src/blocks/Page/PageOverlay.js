import styled from 'styled-components'

const PageOverlay = styled.div`
  position: fixed;
  top: 50px;
  right: 0;
  bottom: 0;
  left: 0;
  background-color: ${props => props.theme.colors.black};
  opacity: 0.5;
  z-index: 9;
`

export default PageOverlay
