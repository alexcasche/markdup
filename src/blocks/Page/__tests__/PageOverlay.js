import PageOverlay from '../PageOverlay'

describe('PageOverlay Element', () => {
  it('renders correctly', () => {
    const tree = render(<PageOverlay />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
