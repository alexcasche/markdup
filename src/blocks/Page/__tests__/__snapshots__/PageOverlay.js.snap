// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageOverlay Element renders correctly 1`] = `
.c0 {
  position: fixed;
  top: 50px;
  right: 0;
  bottom: 0;
  left: 0;
  background-color: #1D1F21;
  opacity: 0.5;
  z-index: 9;
}

<div
  className="c0"
/>
`;
